version.sh

```diff
@@ -0,0 +1,2 @@
+export ARTIFACT_NODE_VERSION='16.14.2'
```

Comment:
Can't we get this from the `.nvmrc`? I feel like we have the Node version hard-coded in too many places now?

Reply:
Yeah, I'll have it pull from the .nvmrc